DigitalOffice/Services/CompanyService/Broker/Requests/InternalGetUserPositionRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Broker.Requests
+{
+    public interface InternalGetUserPositionRequest
+    {
+        public Guid UserId { get; }
+    }
```

Comment:
Use class for broker request or response

Reply:
No-no-no, we use the interfaces. Rewatch last call with Spartak.

Reply:
Exactly, i did not notice)